src/selectors/socket/selectors.js

```diff
@@ -2,42 +2,48 @@ import { createSelector } from 'reselect';
 import { initialState } from '../../reducers/phoenixReducer';
 import { isNullOrEmpty } from '../../utils';
 
-export const selectPhoenix = (state) => state.phoenix || initialState;
-export const selectPhoenixSocket = (state) => state.phoenix.socket;
-export const selectPhoenixSocketDomain = (state) => state.phoenix.domain;
-export const selectPhoenixSocketDetails = (state) => state.phoenix.details;
-export const selectPhoenixSocketAuthenticated = (state) =>
+export const getPhoenix = (state) => state.phoenix || initialState;
```

Comment:
this will be the only breaking changes, the renaming of the selectors. I got this naming from their examples. if you feel we should include this change, im fine by it. But then trixtajs and waitroom will need to update to use the naming convention
[naming convention](https://redux.js.org/usage/deriving-data-selectors#creating-unique-selector-instances)

Reply:
there could be a few. not sure. how can I check this

Reply:
There is a dependency graph in github somewhere. But once released just increment a major version number and make a note on it what was done.